spark-plugins/docs/Tokenizer-sparkcompute.md

```diff
@@ -44,13 +45,12 @@ For example, suppose the tokenizer receives below input records:
     | Spark | Spark /is an engine for /bigdata processing   |
     +=======================================================+
 
-Output schema will contain only a single column "words" having tokenized data in string array form:
-
-    +===============================================+
-    | words                                         |
-    +===============================================+
-    | [hello world, is the, basic application]      |
-    | [hdfs, is a ,file system]                     |
-    | [spark ,is an engine for ,bigdata processing] |
-    +===============================================+
+Output schema will contain additional column "words" having tokenized data in string array form:
 
+    +=====================================================================================================+
+    | topic | sentence                                     | words                                        |
+    +=====================================================================================================+
+    | java  | Hello world / is the /basic application      | [hello world, is the, basic application]     |
```

Comment:
Also, it lowercases everything? Should be sure to document that fact, and perhaps include an option to not lowercase.